x-pack/plugin/esql/qa/testFixtures/src/main/resources/boolean.csv-spec

```diff
@@ -49,7 +49,7 @@ avg(salary):double | always_false:boolean
 
 in
 from employees | keep emp_no, is_rehired, still_hired | where is_rehired in (still_hired, true) | where is_rehired != still_hired;
-
+ignoreOrder:true
```

Comment:
ignoreOrder should only be true; either it's declared and the order is ignored OR it's not and the order count.
I cannot think of scenario where we'd want ignoreOrder:false

Reply:
All the tests are like that now, so we could also throw an exception is a value different from "true" is passed here.